<launch>
      <arg name="vehicleX" default="8" />
      <arg name="vehicleY" default="6" />
      <arg name="vehicleYaw" default="0" />
      <arg name="ns" default="" />
      <!-- 打开地形和扩展地形分析，
  接收：/odom_interface 位姿信息
        /cloud_interface 输出点云
        /map_clearing     可以一键清空地形点云地图，没有发布者，用户自己开发时可使用。
  发布：
        /terrain_map      地形点云地图
  -->
      <include file="$(find terrain_analysis)/launch/terrain_analysis.launch" />
      <!-- 扩展模块：
    - 加入车体中心向外扩展连接片区的墙面边界判断(独立方法), 加入一个局部地形地图分析的半径限制,默认为4m
    - 与基础地形分析一样的订阅+/terrain_map，此地图分析面积更大，频率更低，主要用来生成高质量的全局规划
  发布：
  /terrain_map_ext  扩展地形点云地图 -->
      <!-- <include file="$(find terrain_analysis_ext)/launch/terrain_analysis_ext.launch">
            <arg name="checkTerrainConn" value="true" />
      </include> -->

      <include file="$(find local_planner)/launch/local_planner.launch">
            <arg name="vehicleX" value="$(arg vehicleX)" />
            <arg name="vehicleY" value="$(arg vehicleY)" />
            <arg name="ns" value="$(arg ns)" />
      </include>

</launch>